plugins/evernote.koplugin/main.lua

```diff
@@ -575,25 +575,21 @@ function EvernoteExporter:exportClippings(clippings)
     local msg = "Nothing was exported."
     local all_count = export_count + error_count
     if export_count > 0 and error_count == 0 then
-        if all_count == 1 then
-            msg = T(
-                N_("Exported notes from the book:\n%1",
-                   "Exported notes from the book:\n%1\nand %2 others.",
-                   all_count-1),
-                export_title,
-                all_count-1
-            )
-        end
+        msg = T(
+            N_("Exported notes from the book:\n%1",
+                "Exported notes from the book:\n%1\nand %2 others.",
```

Comment:
Would you mind keeping the indentation as it was for aesthetic reasons? (Not that this isn't logical.)

Reply:
Don't know why but both VS Code and Vim gave me a fight over one character.